cmd/flightctl-pam-issuer/main.go

```diff
@@ -35,65 +34,23 @@ func main() {
 		log.Fatalf("PAM OIDC issuer not configured")
 	}
 
-	ca, _, err := crypto.EnsureCA(cfg.CA)
+	ca, err := crypto.LoadInternalCA(cfg.CA)
 	if err != nil {
-		log.Fatalf("ensuring CA cert: %v", err)
+		log.Fatalf("loading client-signer certificates: %v", err)
 	}
+	caClient := crypto.NewCAClient(cfg.CA, ca)
```

Comment:
Ideally we'd not reuse the API server's client-signer CA, but derive a second intermediate CA from the Flight Control CA that'd be just for signing the PAM issuer's JWT tokens. Can probably wait for later, though.